### DISCUSSION TITLE
Run line of code from a script without cursor jumping ahead

### DISCUSSION THREAD

**Discussion by Btibert3** [posit-dev/positron#4747] [Q&A]
I am using Positron on a Mac (Sonoma 14.6.1) and writing my code in a Python script.  I love the feature that sends the line(s) to the console for evaluation.  It appears that we can do this with Command+Return and Control+Return.  

In both cases, the cursor in the script will jump to the next uncomment command, or the end of the script, which may not always be my desired action.  

My question:  Is there a keyboard shortcut to run the line (or lines selected) without moving the cursor in the script?  I love the jump-ahead feature in general,  but there are also times I would want to be able to send commands without the cursor jumping forward (e.g. Teaching).

**Comment by petetronic** [comment]
Hi @Btibert3, I believe we're tracking this enhancement in https://github.com/posit-dev/positron/issues/2778. Please add your thoughts to that issue to help add to the signal of what matters to users!

**Comment by Btibert3** [comment]
I will comment there, closing this out.
